src/utilities/RobotDealer.cpp

```diff
@@ -95,7 +95,9 @@ int RobotDealer::claimRobotForTactic(RobotType feature, const std::string &roleN
 
         switch (feature) {
 
-        default:return - 1;
+        default:
+            std::cout << "no match for feature" << std::endl;
```

Comment:
It's a warning that the robotdealer can't find a robot with those specifications

Reply:
I know that, but if i would get the error in my terminal I would not know that. 
Something like "[RobotDealer] could not find a robot with the desired properties" Could be better in this case

Besides, you changed behavior so that no robot is claimed (so you maybe don't take a penalty at all, and you hope you don't segfault).